glue/sales_val_flagging.py

```diff
@@ -424,7 +424,7 @@ def get_parameter_df(
     return df_parameters
 
 
-def get_metadata_df(run_id, timestamp, run_type, commit_sha, flagging_hash=""):
```

Comment:
The flagging script is now re-deployed with every commit, so we no longer need to store a reference to its hash; instead, the `commit_sha` should uniquely identify the state of the flagging script at the time that the script was deployed.

One question on this: Do we need to alter the data model at all to remove references to `flagging_hash`?

Reply:
Cool, I double-checked that the script output is correct and added a note to the PR body reminding myself that we need to recreate and recrawl this data after merging 👍🏻